californium-core/src/test/java/org/eclipse/californium/core/coap/RequestTest.java

```diff
@@ -233,7 +233,7 @@ public void testSetOptionsSetsUriHostOption() {
 	@Test
 	public void setObserveFailsForNonGetRequest() {
 
-		Code[] illegalCodes = new Code[]{ Code.DELETE, Code.POST, Code.PUT };
+		Code[] illegalCodes = new Code[]{ Code.PATCH, Code.FETCH, Code.DELETE, Code.POST, Code.PUT };
```

Comment:
According https://tools.ietf.org/html/rfc8132#section-2.4, FETCH is supported for observe.